### ISSUE TITLE
make it easier to mount external storages

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/core#4315]
**TL;DR** There are API limits etc, but we should at least link to documentation.

I just went through the process of mounting ownCloud/WebDAV, Dropbox and Google Drive.

It’s very easy for WebDAV – simple address, username, password.

It’s harder for Dropbox. You need to search how to do it first. Then you probably find @MTGap’s post: http://blog.gapinthecloud.com/2012/06/13/mounting-dropbox-into-owncloud/ – and then you need to create that app. Probably the millionth app which does ownCloud-Dropbox connection. Can’t we have one »official« app which is published and those keys can be prefilled? Maybe even automatically connect. Because there is the possibility that the app is shut down, on erroneous connection it should fall back to expose the input fields of course. What do you think?

The Google Drive connection I didn’t even manage, because searching for client id and client secret and Google Drive yielded no results. How can I do it?

**Comment by MorrisJobke** [comment]
This is covered here http://doc.owncloud.org/server/7.0/admin_manual/configuration/custom_mount_config_gui.html 

@jancborchardt Please reopen if this is rather a problem of linking the documentation inside of ownCloud.

**Comment by MorrisJobke** [comment]
And prefill with our API app for Dropbox for example maybe isn't possible due to API limitations about how often a request is allowed (for example 5000 request per hour or something like that)
